airbyte-integrations/connectors/source-shopify/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Shopify Source CHANGELOG
+
+| Date | Released Version | Notes |
+| :--- | :--- | :--- |
+| `2021-06-07` | `0.1.3` | `Initial Python CDK Airbyte Source Shopify Release` |
+| `2021-06-08` | `0.1.4` | `Added Order Risks Stream` |
```

Comment:
can you place it in reverse order, so most recent should be on top

Reply:
yeah https://github.com/airbytehq/tap-shopify/pull/4 this is for adding more attributes to orders stream.

Reply:
There are 84 attributes across Orders object inside Shopify. All of them are available by default. This is the standard output from API Call, we are not limiting anything here unless we skipping some specific fields, due to `orders.json` schema. 
Additionally, there are: 

- Orders Refunds
- Orders Transactions
- Orders Risks 
- Order Drafts

Those could be fetched using the order_id, for each order_id separately, we do so.

If we missing any attribute of `order.json` schema itself, please verify it using the following documentation on the official site, for availability. [link](https://shopify.dev/docs/admin-api/rest/reference/orders/order#index-2021-04)

For now, the only attribute of Order Stream in general that is not covered is Order Drafts.

Reply:
Found some missing structures inside `order.json`, working to add them.

Reply:
Done.